e2e-tests/gef/cypress/e2e/clone/clone-gef-deal.spec.js

```diff
@@ -1,5 +1,5 @@
 import relative from '../relativeURL';
-import { form, mainHeading, submitButton } from '../partials';
```

Comment:
I dont really like this `form` - we should probably replace this with `cy.clickFooButton`

Reply:
I think this is a wider question -- so nothing to change in this PR. 

It's whether we want to differentiate between submitting a form and just clicking continue button (either way we should look to follow the `cy.` pattern)... I'm of no strong opinion on this